schemas/v1.0/azure.yaml.json

```diff
@@ -27,6 +27,31 @@
                 }
             }
         },
+        "infra": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+                "provider":{
+                    "type": "string",
+                    "title": "Type of infrastructure provisioning provider",
+                    "description": "Optional. When specified will set the infrastructure provisioning provider used to provision the Azure resources for the application. (Default: bicep)",
+                    "enum": [
+                        "bicep",
+                        "terraform"
+                    ]
+                },
+                "path": {
+                    "type": "string",
+                    "title": "Path to the location that contains Azure provisioning templates",
+                    "description": "Optional. When specified will override the path to the Azure provisioning templates for the specified provider. (Default: infra)"
+                },
+                "module": {
```

Comment:
Thinking out loud: is allowing a custom `module` entry-point a useful feature? I wonder if using `main` always is better for consistency.

Reply:
We already support this for the `service` components.  We'll have our convention by default but allows devs to use their own pattern.